core/logging.h

```diff
@@ -39,19 +40,28 @@ static inline void log(
     userMessage += " " + extra;
   }
 
-  std::string message = Port::getHumanTime();
+  // TODO: this is not a cleanest solution, but it works for now
+
+  std::string message;
+
+#ifndef IDF_TARGET
```

Comment:
Maybe change this flag to something more obvious like `ESP_PLATFORM` to be consistent with other cases. Apart from this… i think it can stay like this